var/spack/repos/builtin/packages/star/package.py

```diff
@@ -44,8 +45,12 @@ def edit(self, spec, prefix):
             env["CXXFLAGS_SIMD"] = ""
 
     def build(self, spec, prefix):
+        # different make targets if we're compiling for Mac M1/2
         with working_dir(self.build_directory):
-            make("STAR", "STARlong")
+            if spec.satisfies("target=aarch64:"):
```

Comment:
Given the name I wonder if:
```suggestion
            if spec.satisfies("platform=darwin target=aarch64: "):
```
is better. Which target do you have to make for `linux` on `aarch64`?